_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,111 @@
+This guide explains the end to end flow of creating an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) for your own Connector. Before you start, you should check out our IdentityAttribute introduction first.
+
+<!--- [IdentityAttribute introduction]({_docs_integrate/identityattribute-introduction.md }) --->
+
+## Send Request
+
+To create an IdentityAttribute as a user for your own Connector, you need to send a Request by executing `POST /api/v2/Attributes` with an IdentityAttribute JSON payload:
+
+```jsonc
+{
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<your Connector's Address>",
+    "validFrom": "<start of Attribute validity>",
+    "validTo": "<end of Attribute validity>",
+    "value": {
+      //IdentityAttribute Value
+      "@type": "<type of IdentityAttribute Value>",
+      "<property 1>": <input value 1>, ..., "<property n>": <input value n>
+    },
+    "tags": ["<additional information 1>", ..., "<additional information m>"]
+  }
+}
+```
+
+You need to replace the placeholders marked with <...> appropriately. In particular, it is necessary that you feed one of the available [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) into the payload. Note that the properties `validFrom`, `validTo` and `tags` are optional, so you can omit them.
+
+In general, you can query your Connector's Address by sending a Request using `GET /api/v2/Account/IdentityInfo`. If you use the Demo Connector provided by us for testing purposes in the [interactive excerpt of the Connector's API documentation]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}), the Connector's Address is `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD`.
```

Comment:
I guess here and also later it would be nice link to a use case.

Reply:
I've replaced the "HTTP request using `GET /api/v2/Account/IdentityInfo`" part with a link to a use case now (it's secondary information here anyway). 
Also, I changed the position of the link to the Create an Attribute use case, so that it is in a more prominent place.